i18n/en.po

```diff
@@ -0,0 +1,660 @@
+msgid "RESEND_TOKEN_BUTTON"
+msgstr "Send a new verification code"
+
+#: client/components/mobile-phone-verification/mobile-phone-verification.js:244
+msgid "PHONENUMBER_CHANGE_LABEL"
+msgstr "Wrong phone number?"
+
+#: client/components/mobile-phone-verification/mobile-phone-verification.js:249
+msgid "PHONENUMBER_CHANGE_BUTTON"
+msgstr "Change your phone number"
+
+#: client/components/mobile-phone-verification/mobile-phone-verification.js:254
+msgid "LOGOUT_LABEL"
+msgstr "If you need to cancel the process you can log out below."
+
+#: client/components/mobile-phone-change/mobile-phone-change.js:39
+msgid "MOBILE_PHONE_CHANGE_PAGE_TITLE"
+msgstr "Change mobile number"
+
+#: client/components/mobile-phone-change/mobile-phone-change.js:179
+msgid "CHANGE_PHONENUMBER_BUTTON"
+msgstr "Change your phone number"
+
+#: client/components/mobile-phone-change/mobile-phone-change.js:187
+msgid "CANCEL_BUTTON"
+msgstr "Cancel"
+
+#: client/components/logout/logout.js:23
+msgid "LOGOUT_PAGE_TITLE"
+msgstr "Logout"
+
+#: client/components/logout/logout.js:45
+msgid "LOGOUT_CONTENT"
+msgstr "Logout successful"
+
+#: client/components/logout/logout.js:52
+msgid "LOGIN_AGAIN"
+msgstr "Log in again"
+
+#: client/components/login/login.js:114
+msgid "USERNAME_TITLE"
+msgstr "enter valid email address"
+
+#: client/components/login/login.js:52
+msgid "LOGIN_PAGE_TITLE"
+msgstr "Log in"
+
+#: client/components/login/login.js:273
+#: client/components/login/login.js:277
+#, javascript-format
+msgid "SOCIAL_LOGIN_TEXT ${ link.text }"
+msgstr "Login with ${ link.text }"
+
+#: client/components/login/login.js:329
+msgid "REMEMBER_ME_LABEL"
+msgstr "remember me on this device"
+
+#: client/components/login/login.js:336
+msgid "LOGIN_ADDITIONAL_INFO_TEXT"
+msgstr ""
+"By logging in, you accept the {terms_and_conditions} and the "
+"{privacy_policy} of this WiFi service. "
+
+#: client/components/login/login.js:347
+msgid "LOGIN_TEXT"
+msgstr "Login"
+
+#: client/components/login/login.js:353
+msgid "REGISTER_BUTTON_LABEL"
+msgstr "Not registered yet?"
+
+#: client/components/login/login.js:355
+msgid "REGISTER_BUTTON_TEXT"
+msgstr "Sign up"
+
+#: client/components/header/header.js:114
+#: client/components/header/header.js:127
+#: client/components/header/header.js:182
+#: client/components/header/header.js:195
+#, javascript-format
+msgid "HEADER_LINK ${ link.text }"
+msgstr "${ link.text }"
+
+#: client/components/footer/footer.js:28
+#, javascript-format
+msgid "FOOTER_LINK ${ link.text }"
+msgstr "${ link.text }"
+
+#: client/components/footer/footer.js:37
+msgid "FOOTER_SECONDARY_TEXT"
+msgstr " "
+
+#: client/components/contact-box/contact.js:15
+#, javascript-format
+msgid "CONTACT_SOCIAL_LINK ${ link.alt }"
+msgstr "${ link.alt }"
+
+#: client/components/contact-box/contact.js:20
+msgid "CONTACT_EMAIL_LABEL"
+msgstr "E-mail"
+
+#: client/components/contact-box/contact.js:22
+msgid "CONTACT_EMAIL"
+msgstr "support@openwisp.co"
```

Comment:
:man_facepalming:  :facepalm:  :face_with_head_bandage:  :face_with_thermometer: 
these kind of things should always be configuration... Applies to any other organization specifc detail.

Reply:
I also don't want to define it here, but In configuration, I saw email is also defined for multiple languages i.e. old getText is used in the code for email value also.

Reply:
I am making email for single language only.